tests/unit/h/models/document/_uri_test.py

```diff
@@ -43,10 +43,9 @@ def test_you_cannot_set_content_type_to_null(self, db_session, document_uri):
         with pytest.raises(sa.exc.IntegrityError):
             db_session.flush()
 
-    @pytest.mark.xfail(strict=True)
     def test_unique_constraint_can_take_long_values(self, db_session):
         document_uri = DocumentURI(
-            uri="http://example.com/" * 100000,
+            uri="http://example.com/" * 10000,
```

Comment:
Wrong value from me here on the previous PR

With `100000` it fail first with the previous constraint but then with the plain index on the value.

With `10000` just fails with the old constraint.


We still need a limit on the size of this independently of the constraint.